client/state/plugins/installed/selectors-ts.ts

```diff
@@ -65,6 +71,34 @@ export const getAllPluginsIndexedByPluginSlug = createSelector(
 	]
 ) as { ( state: AppState ): { [ pluginSlug: string ]: Plugin } };
 
+const _filters: { [ key in PluginFilter ]: ( plugin: Plugin ) => boolean } = {
```

Comment:
Before we ship this, let's add a TODO comment, indicating that this almost duplicates `_filters` from `selectors.js` as part of the migration to TS and we're about to remove the duplication soon. 

We might want to add that `_filters.isEqual()` is missing right now but it should be added afterward.

Reply:
I took out `_filters.isEqual()` because as far as I can tell it's never used. It's not a predicate function so it can't be used like the other filters are (i.e. you can't pass it to `Array.filter`).

I added a comment regarding the entire file instead of just `_filters` because there will be other functions that are duplicated during the transition.